tests/pulsar-client-admin-shade-test/pom.xml

```diff
@@ -107,7 +107,6 @@
                         <artifactId>maven-surefire-plugin</artifactId>
                         <configuration>
                             <argLine>${testJacocoAgentArgument} -XX:+ExitOnOutOfMemoryError -Xmx2G -XX:MaxDirectMemorySize=8G
-                                -Dio.netty.leakDetectionLevel=advanced
```

Comment:
do we want to remove this in this PR? (seems like the leak detection is already covered)

Reply:
I'll revisit this. The goal is to use `test.netty.args` for passing leak detection arguments since that's what adds all options that are needed for getting the leak reporting. Without reporting, enabling leak detection is more or less useless since it's not possible to easily notice when something leaks.

Reply:
It's already covered on the next line, there's `${test.additional.args}` which already includes the leak detection settings.